app/Model/LanguageModel.php

```diff
@@ -147,6 +149,7 @@ public function getJsLanguageCode()
             'ca_ES' => 'ca',
             'da_DK' => 'da',
             'de_DE' => 'de',
+            'de_DE_du' => 'de-du',
```

Comment:
The Javascript language should be `de` because jquery-ui doesn't seems to support `de-du`.

Reply:
Should be resolved now